jetty-start/src/main/java/org/eclipse/jetty/start/FileInitializer.java

```diff
@@ -120,35 +124,6 @@ protected Path getDestination(URI uri, String location) throws IOException
         return destination;
     }
 
-    protected void download(URI uri, Path destination) throws IOException
```

Comment:
This was moved to `DownloadFileInitializer` to isolate the behavior to only `MavenLocalRepoFileInitializer` (when downloading from remote repo) and `UriFileInitializer` (for arbitrary URIs).